src/main/scala/com/github/potix2/spark/google/spreadsheets/SparkSpreadsheetService.scala

```diff
@@ -169,14 +151,15 @@ object SparkSpreadsheetService {
     def name: String = sheet.getProperties.getTitle
     lazy val values = {
       val valueRange = context.query(spreadsheet.getSpreadsheetId, name)
-      if ( valueRange.getValues != null )
+      if ( valueRange.getValues != null ) {
```

Comment:
IMHO `Option(valueRange.getValues).fold(...)` will be nicer.

Reply:
You are right! Original types didn't reflect real data. This code was refactored